GentApp/Services/BranchService.cs

```diff
@@ -33,26 +32,40 @@ public async Task<IEnumerable<Branch>> GetBranches()
 			return Enumerable.Empty<Branch>();
 		}
 
-		public async Task<IEnumerable<Branch>> GetBranchesOfCompany(string id) {
-			// TODO: aanpassen
-			HttpResponseMessage response = await HttpClient.GetAsync(apiUrl);
-			if ( response.IsSuccessStatusCode ) {
-				return JsonConvert.DeserializeObject<IEnumerable<Branch>>(await response.Content.ReadAsStringAsync());
-			}
-			return Enumerable.Empty<Branch>();
-		}
-
-		public async Task Save(Branch branch)
+		public async Task<Branch> Save(Branch branch)
 		{
 			try
 			{
-				var response = await HttpClient.PostAsync(apiUrl, new StringContent(JsonConvert.SerializeObject(branch), Encoding.UTF8, "application/json"));
```

Comment:
wrm dit veranderd, is toch goed zo?

Reply:
keb alles hetzelfde gedaan. Zo met eerst je request samenstellen en dan sendasync te gebruiken.

Reply:
cv